evap/staff/forms.py

```diff
@@ -49,10 +72,8 @@ def __init__(self, *args, **kwargs):
         self.fields['course'].choices = choices
 
     def clean(self):
-        if self.cleaned_data['course'] and self.cleaned_data['excel_file']:
-            raise ValidationError('Please select only one of course or Excel file.')
-        if not self.cleaned_data['course'] and not self.cleaned_data['excel_file']:
-            raise ValidationError('Please select either course or Excel File.')
+        if self.cleaned_data['course'] is None:
+            raise ValidationError("Please select a Course from the drop-down.")
```

Comment:
I'd agree but if I change it to use `required=True` it will give an error when using the excel import form even though is_valid is never called on the participant copy form. So going to the course participant import view, selecting an excel file and clicking on "Upload" will cause the upload and test to pass, but after the page reloads it will say "This field is required" on the course selection, which is distracting. Any ideas to circumvent that?